### ISSUE TITLE
Accessibility issues in the GUIs

### ISSUE THREAD

**Issue by Kazaf123** [naman14/Timber#473]
**Description:**
This issue aims to figure out some accessibility issues in the following GUI, and we appreciate if such issues could be fixed, which would bring great assistance for visually impaired users.
 
![image](https://user-images.githubusercontent.com/44084361/161360022-aafaf7ba-4804-4828-9ebf-dc151901506f.png)

**Accessibility Issues:**
1.	The textviews of “专辑” and “艺术家”are relatively small in size.
2.	The button of “more” in each music lacks the content-description to support TalkBack and VoiceOver to read.

**Appreciate:**
Thanks for reading this bug report and I really hope you will repair these issues for people using this App conveniently.
